modules/juce_audio_devices/native/juce_ios_Audio.cpp

```diff
@@ -967,6 +979,7 @@ struct iOSAudioIODevice::Pimpl      : public AudioPlayHead,
         desc.componentFlagsMask = 0;
                 
         setAudioPreprocessingEnabled(isUsingBuiltInSpeaker());
+        setAnalogInputGain(0.5f);

```

Comment:
Should we consider only doing this with headphones or does it need to always be there?

Reply:
This will only apply with devices that have 

> > > It looks like you're somehow pulling in a bunch of changes that I don't think you meant to pull in here. Did you branch off of develop or somewhere else?
> > 
> > 
> > I was pretty sure I branch off mayk-it/JUCE/tree/develop
> 
> Hmm maybe you had some local state from when I was trying to rebase our develop branch :-/... You might need to pull your local develop branch again and then rebase your commit on just that

That make sense. I'll do that